libass/ass.h

```diff
@@ -388,6 +388,16 @@ void ass_renderer_done(ASS_Renderer *priv);
  */
 void ass_set_frame_size(ASS_Renderer *priv, int w, int h);
 
+/**
+ * \brief Set the number of threads to use during rendering. Default is 0,
+ * meaning the number of cores if known, otherwise 1.
+ * \param priv renderer handle
+ * \param threads number of threads
+ *
+ * NOTE: this can only be configured before the first frame is rendered
+ */
+void ass_set_threads(ASS_Renderer *priv, unsigned threads);
+
```

Comment:
This should bump `LIBASS_VERSION` (not the least significant nibble) and adjust the libtool version number *(bump `current` and `age`, reset `revision` iinm)*

Reply:
Libtool docs say to only do this immediately before a "public release", which I _assume_ means a tag?

Reply:
Yes, but astiob suggested([[1]](https://github.com/libass/libass/pull/573#issuecomment-997096831), [[2]](https://github.com/libass/libass/pull/573#issuecomment-997125079)) to always increase the version immediately when a symbol is added or removed since libass is supposedly often used from git master rather than releases.